_sass/minima.scss

```diff
@@ -1,3 +1,5 @@
+// This file is unmodified but must be included so our custom syntax-highlighting partial is read
```

Comment:
I see how that comment does not make sense.  The comment means to say it is unmodified relative to the `minima` theme (the two modifications you see are reverting local change, that were instead moved into main.scss